app/state/reducers/ui/reservationsReducer.js

```diff
@@ -29,6 +29,15 @@ function selectReservationToEdit(state, action) {
   });
 }
 
+function parseError(error) {
+  if (error.response && error.response.non_field_errors && error.response.non_field_errors.length) {
+    return error.response.non_field_errors.join('. ').replace('[\'', '').replace('\']', '');
+  } else if (error.response && error.response.detail) {
+    return error.response.detail;
+  }
+  return 'Jotain meni vikaan';
```

Comment:
Yeah, I think it's okay to have that just in Finnish for the moment. The API error messages are also in Finnish and these are only shown to admins. If we at some point make recurring reservations available to regular users as well, then we might translate these as well.